src/mugs.js

```diff
@@ -790,6 +790,14 @@ define([
         mug.p.conflictedNodeId = null;
     }
 
+    var RESERVED_NAMES = {
```

Comment:
Does it matter what specific kinds of problems the ID can cause? I'd just do a flat list of reserved words and a slightly more generic error, but don't think it matters much either way.

Reply:
80b95c6657763fb32e166a8ba279875ebafa6cff

Fair point about using a generic message. I was trying to preserve the original message (for translations), but this makes the code simpler.